v2/api/containerservice/v1api20210501/managed_cluster_types_gen.go

```diff
@@ -540,13 +540,18 @@ func (cluster *ManagedCluster_Spec) ConvertToARM(resolved genruntime.ConvertToAR
 			}
 			result.Properties.AddonProfiles[key] = *value_ARM.(*ManagedClusterAddonProfile_ARM)
 		}
+	} else { // Set property to empty map, as this resource is set to serialize all collections explicitly
```

Comment:
Comment needs a newline prior so it's on a line by itself.

Reply:
Fixed